dashboard/src/pages/treeConstants.ts

```diff
@@ -0,0 +1,2 @@
+export const DEBOUNCE_INTERVAL = 666;
```

Comment:
I don't think there is any specific reason for this to be a global costat, maybe it could be the default debounce interval for the text input if the user didn't specify any